agents-for-bedrock/use-case-examples/hr-assistant/streamlit/agent_streamlit.py

```diff
@@ -0,0 +1,255 @@
+import io
+import os
+import re
+import time
+import boto3
+import string
+import random
+import streamlit as st
+from datetime import datetime
+
+os.environ['TZ'] = 'US/Pacific'
+time.tzset()
+
+def session_generator():
+    # Generate random characters and digits
+    digits = ''.join(random.choice(string.digits) for _ in range(4))  # Generating 4 random digits
+    chars = ''.join(random.choice(string.ascii_lowercase) for _ in range(3))  # Generating 3 random characters
+
+    # Construct the pattern (1a23b-4c)
+    pattern = f"{digits[0]}{chars[0]}{digits[1:3]}{chars[1]}-{digits[3]}{chars[2]}"
+    print("Session ID: " + str(pattern))
+
+    return pattern
+
+# Bedrock Variable
+agentId = os.environ['BEDROCK_AGENT_ID']
+agentAliasId = os.environ['BEDROCK_AGENT_ALIAS_ID']
+profile_name = os.environ['PROFILE_NAME']
+
+# AWS Session and Clients Instantiation
+agent_client = boto3.Session(profile_name=profile_name).client('bedrock-agent-runtime')
+s3_client = boto3.Session(profile_name=profile_name).client('s3')
+
+# Streamlit CSS
+custom_css = """
+    <style>
+        .text-with-bg {
+        color: white;
+            background-color: #1c2e4a; /* Change this to your desired background color */
+            padding: 10px;
+            border-radius: 5px;
+        }
+
+        .stSpinner {
+            position: fixed;
+            bottom: 20px;
+            z-index: 10;
+        }
+
+        .st-emotion-cache-1ru4d5d, .st-emotion-cache-139wi93 {
+            max-width: 70rem;
+        }
+        .st-emotion-cache-1dp5vir {
+            background-image: linear-gradient(90deg, rgb(148, 241, 246), rgb(148, 189, 246));
+        }
+        .st-emotion-cache-1ghhuty {
+            background-color: #87CEEB;
+        }
+        .st-co, .st-cn, .st-cm, .st-cm {
+            border-bottom-color: rgba(28, 131, 225);
+            border-top-color: rgba(28, 131, 225);
+            border-right-color: rgba(28, 131, 225);
+            border-left-color: rgba(28, 131, 225);
+        }
+    </style>
+"""
+
+# Streamlit App Layout
+st.set_page_config(
+    page_title="Titan Text Premier Demo"
+)
+st.title('Titan Text Premier- HR Assistant')
+st.info("**DISCLAIMER:** This demo uses an Amazon Bedrock foundation model and is not intended to collect any personally identifiable information (PII) from users. Please do not provide any PII when interacting with this demo. The content generated by this demo is for informational purposes only.")
```

Comment:
why are we adding this disclaimer? the impression it gives is that it will collect data when you use the demo. That should not be the case with Bedrock